_posts/2021-01-15-galaxy-wg.md

```diff
@@ -0,0 +1,24 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'New Galaxy Community Organization: Working Groups'
+tags: [galaxy]
+---
+
+<br>
+The Galaxy community is undergoing an __organizational transformation__, with the main focus on improving our global coordination in different areas by defining [__Working Groups__](https://galaxyproject.org/community/wg/). 
```

Comment:
Maybe we can add a motivation why this is needed... Growing community, many areas with different expertise in the community ...